assets/js/modules/analytics/components/settings/SettingsEdit.test.js

```diff
@@ -58,10 +58,12 @@ describe( 'SettingsEdit', () => {
 			},
 		] );
 
-		fetchMock.get( /analytics\/data\/settings/, { body: {} } );
-		fetchMock.get( /analytics-4\/data\/settings/, { body: {} } );
+		fetchMock.get( new RegExp( 'analytics/data/settings' ), { body: {} } );
+		fetchMock.get( new RegExp( 'analytics-4/data/settings' ), {
+			body: {},
+		} );
 
-		fetchMock.get( /\example.com/, {
```

Comment:
Odd, seems like that was actually an error; the `\` here seems unneeded. Thanks for catching that! 👍🏻

Reply:
Shouldn't the `.` be escaped in `.com`?

Reply:
It should have been, yes. But of course it'll still detect `example.com` with `.` operating as a wildcard instead.

It'll actually need to be `new RegExp( 'example\\.com' )`.